soh/soh/OTRGlobals.cpp

```diff
@@ -1166,6 +1165,9 @@ extern "C" void InitOTR() {
 
     SohGui::SetupGuiElements();
     ShipInit::InitAll();
+
+   Rando::StaticData::InitHashMaps();
```

Comment:
```suggestion
    Rando::StaticData::InitHashMaps();
```

Reply:
??? I'm not understanding why the space? That make it misaligned with the line directly above it?

Reply:
Oh weird I looked at it on a different page and now I see it looks misaligned. I'll have a closer look tomorrow.

Reply:
Should be fixed now